### PR TITLE
71: Support for around function without passing tags

### PR DESCRIPTION
**Pull request by TitouanVanBelle**

### CODE REVIEW

File: Cucumberish/Core/CCIBlockDefinitions.h [hunk 1]
```diff
@@ -332,37 +332,48 @@ OBJC_EXTERN void afterTagged(NSArray * tags, CCIScenarioHockBlock afterTaggedBlo
  C function that registers a code block to be used to call the scenario execution block.
  Code blocks registerd with this function will receive two parameters: scenario instance and scenario execution block as a parameter.
  
- If more than one code block matches the scenario, you are still required to call the scenario execution from each registered code block. However, your scenario will be executed once as it is supposed to be.
+ @Note
+ This function should not be called after calling @a beginExecution. 
  
+ @param aroundScenarioBlock code block that will be executed for each scneario, this block receives an instance of the scenario and the scenario execution block.
```
Comment (brentleyjones) [inline Cucumberish/Core/CCIBlockDefinitions.h]: Typo: `scneario`

**Comment by brentleyjones** [comment]
@TitouanVanBelle If we can get some tests for this I think we can move it forward.

**Comment by TitouanVanBelle** [comment]
Yes, I will. Just need to find some time to write them :)

**Comment by TitouanVanBelle** [comment]
Done. Added some tests

**Comment by brentleyjones** [comment]
Thanks @TitouanVanBelle!

**Pull request merged**